### DISCUSSION TITLE
Found fake github.com clone

### DISCUSSION THREAD

**Discussion by gnanet** [community/community#127773] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

hxxps://git.histb.com/ they hide their server with cloudflare, and their domain ownership behind PrivacyGuardian.

the site tries to look like github itself, lot of links point to the original gh-site, but malicious intent is clear to see, beacause "sign in" and "sign up" links are pointing to the clone

![image](https://github.com/community/community/assets/5572007/8e800822-df6a-4e06-8925-29f228480450)

sorry but i could not find any other way to report this kind of threat

**Comment by ExpiredWater** [comment]
You can always report this page to google

https://safebrowsing.google.com/safebrowsing/report_general/

**Reply by gnanet** [reply]
Thank you, i already did report from the search results page. But will do it on the link you suggested. But that may not protect users of Firefox and other browsers, which may not utilize the info from safebrowsing database.

I was just thinking, the content is used from github's domains, so github can block the referrer, to make them more hard to present a clone. Also they could maybe initiate a takedown request at cloudflare, because of the copyright violations.

**Comment by ebndev** [comment]
Hi @gnanet , 👋🏻 We really appreciate you flagging this. I have notified the proper team to look into this.

In the future, the best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
